Models/Round.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Fiks.Models
+{
+    public class Round
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        [StringLength(1000)]
+        public string Descrption { get; set; }
+
+        [Required]
+        public DateTime OpenFrom { get; set; }
+        
+        [Required]
+        public DateTime OpenTo { get; set; }
+
+        [Required]
+        public int Penaliazation { get; set; }
+
+        [Required]
+        public DateTime PenalizationStart { get; set; }
```

Comment:
Je možný, že kolo bude bez penalizace. Chceme to nechat takhle (že to prostě nastavím na stejný datum jako je datum konce), nebo to dát optional? Klidně můžeme nechat takhle, jen jestli na to někdo má nějaký názor :D

Reply:
Je to bodová nebo procentuální penalizace ? Radši bych to měl nullable.

Reply:
Měla by být afaik percentuální. (A asi by bylo fajn to specifikovat v komentáři)

Reply:
cc @MistrMxyzptlk odstraníš required?